miio/toiletlid.py

```diff
@@ -123,22 +139,52 @@ def nozzle_clean(self):
 
     @command(
         click.argument("color", type=EnumType(AmbientLightColor, False)),
+        click.argument("xiaomi_id", type=str, default=""),
         default_output=format_output(
             "Set the ambient light to {color} color the next time you start it."
         ),
     )
-    def set_ambient_light(self, color: AmbientLightColor):
+    def set_ambient_light(self, color: AmbientLightColor, xiaomi_id: str = ""):
```

Comment:
This one and the `bind_xiaomi_band` have a different position for the `xiaomi_id`. Should it be both cases the first argument? Or should it be an option defaulting to `""`, in case that is what works for those who have no `xiaomi_id`?

Reply:
They are the same parameter. `set_ambient_light` if you do not select `xiaomi_id` then set as the default user `""`. But `xiaomi_id` is a required parameter in `bind_xiaomi_band`.

Reply:
Fair enough, let's merge this, it can be improved later on if necessary. Thanks again! :100: